lib/util/Components.js

```diff
@@ -247,6 +247,39 @@ function getWrapperFunctions(context, pragma) {
   ]);
 }
 
+// eslint-disable-next-line valid-jsdoc
```

Comment:
is there no way to have this rule pass with this function's signature?

Reply:
That rule has been deprecated for a long time and it lacks support for lots of typescript syntaxes.

Reply:
gotcha; so we'd have to use a similar rule from the typescript plugin?

Reply:
You are looking for eslint-plugin-jsdoc .

Reply:
typescript plugin does not handle jsdoc as far as I know